core/station/api/spec.did

```diff
@@ -829,6 +863,8 @@ type ListRequestsOperationType = variant {
   RemoveUserGroup;
   // An operation for changing a canister.
   ChangeCanister;
+  // An operation for changing a managed canister.
+  ChangeManagedCanister;
```

Comment:
Ideally you would be able to filter here by any or a specific canister id, similar to what we have in the transfer on line 843